Extensions/TopDownMovementBehavior/topdownmovementruntimebehavior.ts

```diff
@@ -18,18 +23,26 @@ namespace gdjs {
     _rotateObject: any;
     _angleOffset: any;
     _ignoreDefaultControls: any;
+    _viewpoint: any;
+    _customIsometryAngle: any;
 
     /** The latest angle of movement, in degrees. */
     _angle: float = 0;
 
     //Attributes used when moving
+    _x: number = 0;
+    _y: number = 0;
     _xVelocity: float = 0;
     _yVelocity: float = 0;
     _angularSpeed: number = 0;
     _leftKey: boolean = false;
     _rightKey: boolean = false;
     _upKey: boolean = false;
     _downKey: boolean = false;
+    
+    // @ts-ignore
+    _basisTransformation: BasisTransformation;
+    _point: number[] = [0, 0];
```

Comment:
Prefer `float` instead of `number` everywhere (or `integer` if you manipulate integers).

I recommend also defining a `type FloatPoint = [float, float];`, this is done in other extensions and will be probably used more in the future :)

Reply:
This seems a temporary variable, let's either define it in the function using it or changing its name to identify that it's "a temporary just here to avoid allocations when computing transformations" (so `_temporaryPointForTransformations`).